woodwork/utils.py

```diff
@@ -54,6 +54,36 @@ def import_or_none(library):
         return None
 
 
+def ww_concat(base, to_add):
```

Comment:
can we add specific tests here that are ensuring this function does as we'd expect

Reply:
will do, adding to `test_utils.py`